app/controllers/AdditionalMetadataClosureStatusController.scala

```diff
@@ -78,9 +79,9 @@ class AdditionalMetadataClosureStatusController @Inject() (
     implicit request: Request[AnyContent] =>
       val errorFunction: Form[ClosureStatusFormData] => Future[Result] = { formWithErrors: Form[ClosureStatusFormData] =>
         for {
-          (consignmentRef, filePaths, parentFolderId) <- cache.getOrElseUpdate[(String, List[String], UUID)](s"$consignmentId-data", 1.hour)(
+          (consignmentRef, filePaths, closureProperties) <- cache.getOrElseUpdate(s"$consignmentId-data", 1.hour)(
             for {
-              details <- consignmentService.getConsignmentDetails(consignmentId, request.token.bearerAccessToken)
+              closureProperties <- displayPropertiesService.getDisplayProperties(consignmentId, request.token.bearerAccessToken, Some(metadataType)).map(_.map(_.summary))
```

Comment:
You're setting this in the cache at line 55, so maybe this should be used?

Reply:
Yeah, we are using the cache at line 82.